.travis.yml

```
@@ -11,7 +13,7 @@ before_install:
   - curl https://raw.githubusercontent.com/hmcts/reform-api-docs/master/bin/publish-swagger-docs-group-dockerless.sh > publish-swagger-docs.sh
 cache:
   directories:
-  - $HOME/.gradle/caches/
+  - $HOME/.gradle/caches/h
```

:shipit: this intended?
```suggestion
  - $HOME/.gradle/caches/h
```

---

Hi @timja - That was by mistake. I removed that.